File: 0682/sesión10 - soporte MSSQL/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/MovimientosDTO.java
package pe.edu.pucp.softinv.model;

import java.util.Date;

public class MovimientosDTO {

    private Integer movimientoId;
    private TiposMovimientosDTO tipoMovimiento;
    private Date fechaMovimiento;
    private Date fechaCreacion;
    private TiposDocumentosDTO tipoDocumento;
    private String numeroDocumento;
    private AlmacenesDTO almacen;
    private ProductosDTO producto;
    private Integer cantidad;
    private Double valor;

    public MovimientosDTO() {
        this.movimientoId = null;
        this.tipoMovimiento = null;
        this.fechaMovimiento = null;
        this.fechaCreacion = null;
        this.tipoDocumento = null;
        this.numeroDocumento = null;
        this.almacen = null;
        this.producto = null;
        this.cantidad = null;
        this.valor = null;
    }

    public MovimientosDTO(Integer movimientoId, TiposMovimientosDTO tipoMovimiento, Date fechaMovimiento, Date fechaCreacion, TiposDocumentosDTO tipoDocumento, String numeroDocumento, AlmacenesDTO almacen, ProductosDTO producto, Integer cantidad, Double valor) {
        this.movimientoId = movimientoId;
        this.tipoMovimiento = tipoMovimiento;
        this.fechaMovimiento = fechaMovimiento;
        this.fechaCreacion = fechaCreacion;
        this.tipoDocumento = tipoDocumento;
        this.numeroDocumento = numeroDocumento;
        this.almacen = almacen;
        this.producto = producto;
        this.cantidad = cantidad;
        this.valor = valor;
    }

    /**
     * @return the movimientoId
     */
    public Integer getMovimientoId() {
        return movimientoId;
    }

    /**
     * @param movimientoId the movimientoId to set
     */
    public void setMovimientoId(Integer movimientoId) {
        this.movimientoId = movimientoId;
    }

    /**
     * @return the tipoMovimiento
     */
    public TiposMovimientosDTO getTipoMovimiento() {
        return tipoMovimiento;
    }

    /**
     * @param tipoMovimiento the tipoMovimiento to set
     */
    public void setTipoMovimiento(TiposMovimientosDTO tipoMovimiento) {
        this.tipoMovimiento = tipoMovimiento;
    }

    /**
     * @return the fechaMovimiento
     */
    public Date getFechaMovimiento() {
        return fechaMovimiento;
    }

    /**
     * @param fechaMovimiento the fechaMovimiento to set
     */
    public void setFechaMovimiento(Date fechaMovimiento) {
        this.fechaMovimiento = fechaMovimiento;
    }

    /**
     * @return the fechaCreacion
     */
    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    /**
     * @param fechaCreacion the fechaCreacion to set
     */
    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }

    /**
     * @return the tipoDocumento
     */
    public TiposDocumentosDTO getTipoDocumento() {
        return tipoDocumento;
    }

    /**
     * @param tipoDocumento the tipoDocumento to set
     */
    public void setTipoDocumento(TiposDocumentosDTO tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    /**
     * @return the numeroDocumento
     */
    public String getNumeroDocumento() {
        return numeroDocumento;
    }

    /**
     * @param numeroDocumento the numeroDocumento to set
     */
    public void setNumeroDocumento(String numeroDocumento) {
        this.numeroDocumento = numeroDocumento;
    }

    /**
     * @return the almacen
     */
    public AlmacenesDTO getAlmacen() {
        return almacen;
    }

    /**
     * @param almacen the almacen to set
     */
    public void setAlmacen(AlmacenesDTO almacen) {
        this.almacen = almacen;
    }

    /**
     * @return the producto
     */
    public ProductosDTO getProducto() {
        return producto;
    }

    /**
     * @param producto the producto to set
     */
    public void setProducto(ProductosDTO producto) {
        this.producto = producto;
    }

    /**
     * @return the cantidad
     */
    public Integer getCantidad() {
        return cantidad;
    }

    /**
     * @param cantidad the cantidad to set
     */
    public void setCantidad(Integer cantidad) {
        this.cantidad = cantidad;
    }

    /**
     * @return the valor
     */
    public Double getValor() {
        return valor;
    }

    /**
     * @param valor the valor to set
     */
    public void setValor(Double valor) {
        this.valor = valor;
    }
}

File: 0682/sesión11 - back-end C#/SoftInv/SoftInvModel/MovimientosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.Model
{
    public class MovimientosDTO
    {
        private int? movimientoId;
        private TiposMovimientosDTO tipoMovimiento;
        private DateTime? fechaMovimiento;
        private DateTime? fechaCreacion;
        private TiposDocumentosDTO tipoDocumento;
        private string numeroDocumento;
        private AlmacenesDTO almacen;
        private ProductosDTO producto;
        private int? cantidad;
        private double? valor;

        public MovimientosDTO()
        {
            this.MovimientoId = null;
            this.TipoMovimiento = null;
            this.FechaMovimiento = null;
            this.FechaCreacion = null;
            this.TipoDocumento = null;
            this.NumeroDocumento = null;
            this.Almacen = null;
            this.Producto = null;
            this.Cantidad = null;
            this.Valor = null;
        }

        public MovimientosDTO(int movimientoId, TiposMovimientosDTO tipoMovimiento, DateTime fechaMovimiento, DateTime fechaCreacion, TiposDocumentosDTO tipoDocumento, string numeroDocumento, AlmacenesDTO almacen, ProductosDTO producto, int cantidad, double valor)
        {
            this.MovimientoId = movimientoId;
            this.TipoMovimiento = tipoMovimiento;
            this.FechaMovimiento = fechaMovimiento;
            this.FechaCreacion = fechaCreacion;
            this.TipoDocumento = tipoDocumento;
            this.NumeroDocumento = numeroDocumento;
            this.Almacen = almacen;
            this.Producto = producto;
            this.Cantidad = cantidad;
            this.Valor = valor;
        }

        public int? MovimientoId { get => movimientoId; set => movimientoId = value; }
        public TiposMovimientosDTO TipoMovimiento { get => tipoMovimiento; set => tipoMovimiento = value; }
        public DateTime? FechaMovimiento { get => fechaMovimiento; set => fechaMovimiento = value; }
        public DateTime? FechaCreacion { get => fechaCreacion; set => fechaCreacion = value; }
        public TiposDocumentosDTO TipoDocumento { get => tipoDocumento; set => tipoDocumento = value; }
        public string NumeroDocumento { get => numeroDocumento; set => numeroDocumento = value; }
        public AlmacenesDTO Almacen { get => almacen; set => almacen = value; }
        public ProductosDTO Producto { get => producto; set => producto = value; }
        public int? Cantidad { get => cantidad; set => cantidad = value; }
        public double? Valor { get => valor; set => valor = value; }
    }
}
